packages/rolldown/test/cases/external/array/config.ts

```diff
@@ -1,4 +1,4 @@
-import type { RollupOptions, RollupOutput } from 'rolldown'
+import type { RollupOptions } from '../../../../src'
```

Comment:
Because of pnpm, rolldown was no longer able to reference itself, so had to use relative paths.